### PR TITLE
Exec copy march 25 2022

### PR DESCRIPTION
**Pull request by prose11**
Thanks for the review @blimpa :)

### CODE REVIEW

File: governance/votes/Executive vote - March 25, 2022.md [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+
+Unless otherwise noted, the changes and additions listed above are subject to the [GSM Pause Delay](https://manual.makerdao.com/parameter-index/core/param-gsm-pause-delay). This means that if this executive proposal passes, the changes and additions listed above will only become active in the Maker Protocol after the GSM Pause Delay has expired. The GSM Pause Delay is currently set to **48 hours**.
+
+If this executive proposal does not pass within 30 days, then it will expire and can no longer have any effect on the Maker Protocol.
+
+---
+
+## Proposal Details
+
+### Emergency Shutdown Authorization
```
Comment (blimpa) [inline governance/votes/Executive vote - March 25, 2022.md:34]: "Authorizing ESM on DssFlash to remove Governance Permissions"?

File: governance/votes/Executive vote - March 25, 2022.md [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+The Governance Facilitator(s) and the Protocol Engineering Core Unit have placed an executive proposal into the voting system. MKR Holders should vote for this proposal if they support the following alterations to the Maker Protocol.
+
+If you are new to voting in the Maker Protocol, please see the [voting guide](https://community-development.makerdao.com/en/learn/governance/how-voting-works/) to learn how voting works, and this [wallet setup guide](https://community-development.makerdao.com/en/learn/governance/voting-setup/) to set up your wallet to vote.
+
+---
+
+## Executive Summary
+
+If this executive proposal passes, the following **changes** will occur within the Maker Protocol:
+- The [Emergency Shutdown Module](https://makerdao.world/en/learn/governance/emergency-shutdown) will be authorized to revoke Governance control over the [Flash Mint Module](https://manual.makerdao.com/module-index/module-flash-mint-module) after an emergency shutdown.
```
Comment (blimpa) [inline governance/votes/Executive vote - March 25, 2022.md:19]: Maybe "over the Flash Mint Module (DssFlash)"

**Review by blimpa — CHANGES_REQUESTED** [review]

File: governance/votes/Executive vote - March 25, 2022.md [hunk 3]
```diff
@@ -0,0 +1,63 @@
+---
+title: Template - [Executive Vote] Authorizing the ESM on DssFlash and Support for TUSD Implementation - March 25, 2022
+summary: Authorizing the Emergency Security Module on DssFlash to revoke governance control after Emergency shutdown and Adding Support for the New TUSD Implementation.
```
[the remark this answers is not in the document]
Comment (prose11) [reply governance/votes/Executive vote - March 25, 2022.md]: Yeah was a bit of a mess good call, made it more like a sentence should be :)

File: governance/votes/Executive vote - March 25, 2022.md [hunk 4]
```diff
@@ -0,0 +1,63 @@
...
+date: 2022-03-25T00:00:00.000Z
+address: "$spell_address"
+
+---
+# [Executive Proposal] Support for TUSD Implementation and Authorizing the ESM on DssFlash - March 25, 2022
+
+The Governance Facilitator(s) and the Protocol Engineering Core Unit have placed an executive proposal into the voting system. MKR Holders should vote for this proposal if they support the following alterations to the Maker Protocol.
+
+If you are new to voting in the Maker Protocol, please see the [voting guide](https://community-development.makerdao.com/en/learn/governance/how-voting-works/) to learn how voting works, and this [wallet setup guide](https://community-development.makerdao.com/en/learn/governance/voting-setup/) to set up your wallet to vote.
```
[the remark this answers is not in the document]
Comment (prose11) [reply governance/votes/Executive vote - March 25, 2022.md:12]: Yeah I wouldn't sweat it too much, we can look into changing the template tho to be a bit more precise

File: governance/votes/Executive vote - March 25, 2022.md [hunk 5]
```diff
@@ -0,0 +1,63 @@
...
+
+If this executive proposal passes, the following **additions** will be made to the Maker Protocol:
+- Support for the new TUSD implementation contract ([0xd8D59c59Ab40B880b54C969920E8d9172182Ad7b](https://etherscan.io/address/0xd8d59c59ab40b880b54c969920e8d9172182ad7b)), which will allow collateral operation of TUSD-A to resume.
+
+**Voting for this executive proposal will place your MKR in support of the changes and additions outlined above.**
+
+Unless otherwise noted, the changes and additions listed above are subject to the [GSM Pause Delay](https://manual.makerdao.com/parameter-index/core/param-gsm-pause-delay). This means that if this executive proposal passes, the changes and additions listed above will only become active in the Maker Protocol after the GSM Pause Delay has expired. The GSM Pause Delay is currently set to **48 hours**.
+
+If this executive proposal does not pass within 30 days, then it will expire and can no longer have any effect on the Maker Protocol.
```
[the remark this answers is not in the document]
Comment (prose11) [reply governance/votes/Executive vote - March 25, 2022.md]: Fixed, remind me to submit a change to the template :)

File: governance/votes/Executive vote - March 25, 2022.md [hunk 6]
```diff
@@ -0,0 +1,63 @@
...
+
+This change is intended to patch a vulnerability discovered in the interaction [between the Emergency Shutdown Module and the Flash Mint Module](https://forum.makerdao.com/t/flash-loan-debt-manipulation-during-emergency-shutdown-vulnerability-postmortem/14036). Specifically, this change prevents a malicious governance action from re-activating the DssFlash module after the [FlashKiller contract](https://github.com/makerdao/flash-killer) disables it.
+
+### Adding Support for New TUSD Implementation 
+
+Based on [this post](https://forum.makerdao.com/t/2022-03-25-adding-support-for-the-new-tusd-implementation-address/14189) from the [Protocol Engineering Core Unit](https://mips.makerdao.com/mips/details/MIP38#protocol-engineering-pe-001-), support for the new TUSD implementation address will be added if this executive proposal passes.
+
+This addition is intended to resume the collateral operations for TUSD-A, which automatically halted after the [February 24th change](https://etherscan.io/tx/0x125823f2914e4f14e06b9b1b30fe9dd9512b36354cc1f6c063118c4fe03d8287) from the TUSD team, pointing to a new implementation address.
```
[the remark this answers is not in the document]
Comment (prose11) [reply governance/votes/Executive vote - March 25, 2022.md]: "pointed" should do the trick

File: governance/votes/Executive vote - March 25, 2022.md (hunk 1 above)
Comment (prose11) [reply governance/votes/Executive vote - March 25, 2022.md]: Good call

**Comment by prose11** [comment]
Thanks Pablo :)

**Pull request merged**